thirdparty_builtin/CMakeLists.txt

```diff
@@ -132,8 +132,9 @@ if(ENABLE_BENCHMARKS)
           message(FATAL_ERROR "Google Benchmark cannot be built when BUILD_SHARED_LIBS=ON or on Windows")
         endif()
 
-        add_subdirectory(gbenchmark-master-2017-05-19
-                         ${BLT_BUILD_DIR}/thirdparty_builtin/gbenchmark-master-2017-05-19)
+        set(BENCHMARK_ENABLE_TESTING OFF CACHE "" BOOL)
```

Comment:
I think that you should check the BLT_CXX_STD string and CMAKE_CXX_STANDARD here and at least issue a warning.

Reply:
Will CMAKE_CXX_STANDARD cover the case where the compiler defaults to a high enough standard?  I don't want to throw an error if it was going to succeed without user intervention.

Reply:
No, which is why I said it should issue a warning. A use could also directly add the correct flag to `CMAKE_CXX_FLAGS`, and we wouldn't want that to fail.

Reply:
Warning added.